### DISCUSSION TITLE
ANN: new feature -- searchable emoji list

### DISCUSSION THREAD

**Discussion by williamstein** [sagemathinc/cocalc#6430] [Announcements]
In CoCalc anywhere you use the WYSIWYG Slate-based rich text Markdown editor, you can type : and search from thousands of standard emojis.  Here it is in action in a Jupyter notebook:

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/1276278/218295284-293d9a5b-63ed-4f47-890e-d67afc67ff3d.png">

When you first type ":" you'll see the dropdown in a fraction of the second, showing the most popular emojis.  Type further, and you'll instead do a full text search through a large number of matching emojis.   These emojis are standard (provided by markdown-it) and will also work in Github, etc.

**Comment by williamstein** [comment]
**Exercise:** How many emojis are there in CoCalc whose name has "mountain" in it?  ⛰️

**Comment by Blaec-CoCalc** [comment]
8, but 9 if you count from Mount instead! (Mount Fuji included.)

**Comment by williamstein** [comment]
I put most of the heart emojis in this valentine's day plot: https://cocalc.com/wstein/support/valentines-day/files/heart.ipynb

**Comment by williamstein** [comment]
UPDATE: I changed this so you have to type at least one character after the initial ":".  This is different than Github, but it is MUCH less annoying.
